packages/frontend/src/utils/account.test.js

```diff
@@ -0,0 +1,20 @@
+import { shortenAccountId } from './account';
+
+test('ID is shorten with correct formats', () => {
```

Comment:
It's easier to read if test cases are written with "should" in them:
As an example "It should shorten account ID with correct format"

Btw, great work covering it with tests 🤟